File: C++ Programming/problems on bitwise operator/problem12.cpp
/*

	12. Write a program which accept one number and position from user and off that bit. Return modified number. 
	
	Input :  10 2
	Output :  8 

	Author : Milind Nalawade
	
	Date : 03/08/2021
	
	Language : C++ programming
	
*/
using namespace std;
#include<iostream>
#include<stdlib.h>

//Logic 
int ChkBit(int no,int pos)
{
	int flag=0;
	flag=no^pos;
	return flag;		
}

//Entry point function 
int main(int argc,char * argv[])
{
	int no=atoi(argv[1]);
	int pos=atoi(argv[2]);
	int ret=ChkBit(no,pos);
	cout<<"\n Output :"<<ret<<"\n";
	return 0;
}	

File: Python Programming/problems on bitwise operator/problem12.py
'''
12. Write a program which accept one number and position from user and off that bit. Return modified number. 
	
	Input :  10 2
	Output :  8 
	
	Author : Milind Nalawade
	
	Date : 03/08/2021
	
	Language : Python 3.9.0

'''
import sys

#====
#Logic
#====
def ChkBit(no,pos):
	flag=0
	flag=no^pos
	return flag;
		
#=============
#Entry point function
#=============
def main():
	no=int(sys.argv[1])
	pos=int(sys.argv[2])
	ret=int(ChkBit(no,pos))
	print("\n Output : {} \n".format(ret));	
#=====
#starter
#=====	
if __name__=="__main__":
	main()